src/value.rs

```diff
@@ -92,9 +96,12 @@ use gobject_ffi;
 /// Some types (e.g. `String` and objects) support `None` values while others
 /// (e.g. numeric types) don't.
 ///
+/// `Value` does not implement the `Send` trait, but [`SendValue`](struct.SendValue.html) can be
+/// used instead.
+///
 /// See the [module documentation](index.html) for more details.
 #[repr(C)]
-pub struct Value(gobject_ffi::GValue);
+pub struct Value(gobject_ffi::GValue, PhantomData<*const c_void>);
```

Comment:
@sdroege What role does PhantomData?

Reply:
See commit message :) It makes the struct non-Send. Without having anything inside the struct that is not Send, the compiler will implicitely derive Send.

Reply:
Missed this, Thanks.

Reply:
`impl !Sync for T {}` still nightly-only, maybe this need TODO?

Reply:
Ok